Makefile

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+# See LICENSE.txt for license information.
+
+## Docker Build Versions
+DOCKER_BUILD_IMAGE = golang:1.15.7
+DOCKER_BASE_IMAGE = alpine:3.13
+
+# Variables
+GO = go
+APP := rotator
+APPNAME := node-rotator
+FLEET_CONTROLLER_IMAGE ?= mattermost/node-rotator:test
+
+# Build variables
+COMMIT_HASH ?= $(shell git rev-parse HEAD)
+BUILD_DATE  ?= $(shell date +%FT%T%z)
```

Comment:
Declaring this values which are useful for tagging the image, but I am not seeing them used, unless I missed anything.

Reply:
You are right, we may use in the future but not needed now. Removed thanks :)